src/main/java/com/joseeduardo/todoist/service/impl/TarefaServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.joseeduardo.todoist.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import com.joseeduardo.todoist.entity.TarefaEntity;
+import com.joseeduardo.todoist.entity.UsuarioEntity;
+import com.joseeduardo.todoist.mapper.TarefaMapper;
+import com.joseeduardo.todoist.model.CriarTarefaEntradaDTO;
+import com.joseeduardo.todoist.repository.TarefaRepository;
+import com.joseeduardo.todoist.service.TarefaService;
+import com.joseeduardo.todoist.service.UsuarioService;
+
+@Service
+public class TarefaServiceImpl implements TarefaService {
+
+	@Autowired
+	private TarefaRepository tarefaRepository;
+	@Autowired
+	private UsuarioService usuarioService;
+
```

Comment:
Pode dar uns espaços

```suggestion
	@Autowired
	private TarefaRepository tarefaRepository;
	
	@Autowired
	private UsuarioService usuarioService;

```